manifests/service.pp

```diff
@@ -0,0 +1,31 @@
+# configure, enable, and start pulpcore services
+# @api private
+class pulpcore::service {
+
+  systemd::unit_file { 'pulpcore-api.service':
+    content => template('pulpcore/pulpcore-api.service.erb'),
+    active  => true,
+    enable  => true,
+  }
+
+  systemd::unit_file { 'pulpcore-content.service':
+    content => template('pulpcore/pulpcore-content.service.erb'),
+    active  => true,
+    enable  => true,
+  }
+
+  systemd::unit_file { 'pulpcore-resource-manager.service':
+    content => template('pulpcore/pulpcore-resource-manager.service.erb'),
+    active  => true,
+    enable  => true,
+  }
+
+  systemd::unit_file { 'pulpcore-worker@.service':
+    content => template('pulpcore/pulpcore-worker@.service.erb'),
+  } ->
```

Comment:
Linting doesn't like this style, but in most modules we ignore this. You can ignore it, but you can also add an explicit require as a property to the service. I think that needs to happen anyway: `require => [Systemd::Unit_file['pulpcore-worker@.service'], Class['systemd::systemctl::daemon_reload']]`

Reply:
With the require you can now drop the `->` here. I'd also add a newline after it.